front50-core/src/test/groovy/com/netflix/spinnaker/front50/model/plugins/PluginInfoServiceSpec.groovy

```diff
@@ -105,12 +109,20 @@ class PluginInfoServiceSpec extends Specification {
 
     PluginInfo.Release newRelease = new PluginInfo.Release(version: "2.0.0")
 
+    and:
+    PluginInfo originalPluginInfo = new PluginInfo(id: "foo.bar")
```

Comment:
I had to return the `originalPluginInfo` the second time `findById` was called for the delta detection to work.  This mimics fetching the pristine value from the data store.